### DISCUSSION TITLE
Nice job on the latest firmware - plus a question.

### DISCUSSION THREAD

**Discussion by cdw2000** [ElectroTechnique/electrotechnique.github.io#55] [New developments]
Hey, I just wanted to give a shout out for the latest updates.   The exponential attack and decay curves on the envelopes sound great!   Very smooth sound decay even at maximum time setting.

I also like the expanded filter resonance range.

One question - I was looking in the Master + MT branch and I saw a commit to add exponential glide.   Did that make it into V2.20/2.21 as well?

**Comment by Electrotechnique** [comment]
No, I was planning to just push out Master + MT branch as master fairly soon, because there are considerable improvements to memory usage due to the refactoring. It's still being worked on, we're hoping to introduce multi-timbrality - but the design of the code is beyond what I can manage, so others are working on it.

**Comment by cdw2000** [comment]
Yay!   V2.3 adds exponential glide!   Thanks to all for doing this.
